### ISSUE TITLE
Add Backend repo link to ReadMe

### ISSUE THREAD

**Issue by Umoren** [johnotu/african-food-react-app#7]
This is not an issue but it'll be great if https://github.com/johnotu/african-food-node-server.git was added to the documentation so future contributors would know how to contribute to this.
Thanks.

**Comment by johnotu** [comment]
Yeah, that'll help. While doing that we can replace the default React README with one that properly describes the project. Looking forward to a PR.
